x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/create_rules_route.ts

```diff
@@ -26,7 +26,7 @@ import { convertCreateAPIToInternalSchema } from '../../schemas/rule_converters'
 export const createRulesRoute = (
   router: SecuritySolutionPluginRouter,
   ml: SetupPlugins['ml'],
-  ruleDataClient?: RuleDataClient | null
+  ruleDataClient?: RuleDataClient | null // TODO: Us this for RAC (otherwise delete it)
```

Comment:
If still relevant, otherwise delete?

```suggestion
  ruleDataClient?: RuleDataClient | null // TODO: Use this for RAC (otherwise delete it)
```